plexapi/config.py

```diff
@@ -42,24 +42,46 @@ def get(self, key, default=None, cast=None):
     def _asDict(self):
         """ Returns all configuration values as a dictionary. """
         config = defaultdict(dict)
+        config.update(self._defaults())
         for section in self._sections:
             for name, value in self._sections[section].items():
                 if name != '__name__':
                     config[section.lower()][name.lower()] = value
         return dict(config)
 
+    def _defaults(self):
+        from uuid import getnode
+        from platform import uname
+        from plexapi import PROJECT, VERSION
+
+        platform_name, device_name, platform_version = uname()[0:3]
```

Comment:
Fix the slicing.

Reply:
Explain fix how? fix what? what is wrong with the current code?

Reply:
why is the current slicing wrong. how is the correct way?